DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes/src/CMakeLists.txt

```
@@ -6,12 +6,21 @@ set(FPGA_TARGET ${TARGET_NAME}.fpga)
 
 # FPGA board selection
 if(NOT DEFINED FPGA_DEVICE)
-    set(FPGA_DEVICE "Agilex")
+    set(FPGA_DEVICE "Agilex7")
```

```suggestion
    set(FPGA_DEVICE "Agilex")
```

Let's have this Agilex7 renaming in a separate PR: https://github.com/oneapi-src/oneAPI-samples/pull/1543